code/game/objects/structures/tables_racks.dm

```diff
@@ -226,6 +226,9 @@
 		deconstruct(TRUE)
 	return ITEM_INTERACT_SUCCESS
 
+/obj/structure/table/analyzer_act(mob/living/user, obj/item/analyzer/tool, list/modifiers)
+	return table_place_act(user, tool, modifiers)
```

Comment:
We shouldn't need this anymore since tables also have `TRAIT_COMBAT_MODE_SKIP_INTERACTION`

Reply:
Didn't know that. Yup